includes/gateways/stripe/includes/class-give-stripe-customer.php

```diff
@@ -374,7 +374,8 @@ public function create_customer() {
 				__( 'Stripe - Customer Creation Error', 'give' ),
 				$e->getMessage()
 			);
-
+			give_set_error( 'stripe_error', $e->getMessage() );
+			give_send_back_to_checkout( "?payment-mode={$payment_mode}&form_id={$post_data['post_data']['give-form-id']}" );
```

Comment:
Should we set this string to a variable and then pass it in, for ease of potential debugging with XDebug or other tools?

Reply:
I think if you set the breakpoint to the line number you'll be able to debug it just as you would a variable.